plugins/Types/ClassDiagram/edu.toronto.cs.se.modelepedia.classdiagram/model/classdiagram.ecore_diagram

```diff
@@ -395,35 +429,145 @@
+    <element xmi:type="ecore:EReference" href="classdiagram.ecore#//Class/compositionsAsComposite"/>
+    <bendpoints xmi:type="notation:RelativeBendpoints" xmi:id="_rmWo4jwjEeiDnaL1jp_puA" points="[0, 0, 0, 0]$[0, 0, 0, 0]"/>
+  </edges>
+  <edges xmi:type="notation:Connector" xmi:id="_rmYeEDwjEeiDnaL1jp_puA" type="4002" source="_cTNho0YgEeeaV4gKFnG5PA" target="_TiGMICk_EeixYJV0QD8XrQ">
+    <children xmi:type="notation:DecorationNode" xmi:id="_rmZFIDwjEeiDnaL1jp_puA" type="6001">
+      <layoutConstraint xmi:type="notation:Location" xmi:id="_rmZFITwjEeiDnaL1jp_puA" x="-13" y="-9"/>
+    </children>
+    <children xmi:type="notation:DecorationNode" xmi:id="_rmZFIjwjEeiDnaL1jp_puA" type="6003">
+      <layoutConstraint xmi:type="notation:Location" xmi:id="_rmZFIzwjEeiDnaL1jp_puA" x="-18" y="15"/>
+    </children>
+    <styles xmi:type="notation:FontStyle" xmi:id="_rmYeETwjEeiDnaL1jp_puA" fontName="Ubuntu"/>
+    <element xmi:type="ecore:EReference" href="classdiagram.ecore#//Operation/parameterTypes"/>
+    <bendpoints xmi:type="notation:RelativeBendpoints" xmi:id="_rmYeEjwjEeiDnaL1jp_puA" points="[0, 0, 0, 0]$[0, 0, 0, 0]"/>
+  </edges>
+  <edges xmi:type="notation:Connector" xmi:id="_rma6UDwjEeiDnaL1jp_puA" type="4002" source="_ef9IYClhEeikePo-S06NPw" target="_cTLscEYgEeeaV4gKFnG5PA">
+    <children xmi:type="notation:DecorationNode" xmi:id="_rma6UzwjEeiDnaL1jp_puA" type="6001">
+      <layoutConstraint xmi:type="notation:Location" xmi:id="_rma6VDwjEeiDnaL1jp_puA" y="40"/>
+    </children>
+    <children xmi:type="notation:DecorationNode" xmi:id="_rma6VTwjEeiDnaL1jp_puA" type="6003">
+      <layoutConstraint xmi:type="notation:Location" xmi:id="_rma6VjwjEeiDnaL1jp_puA" y="60"/>
+    </children>
+    <styles xmi:type="notation:FontStyle" xmi:id="_rma6UTwjEeiDnaL1jp_puA" fontName="Ubuntu"/>
+    <element xmi:type="ecore:EReference" href="classdiagram.ecore#//Composition/constituent"/>
+    <bendpoints xmi:type="notation:RelativeBendpoints" xmi:id="_rma6UjwjEeiDnaL1jp_puA" points="[0, 0, 0, 0]$[0, 0, 0, 0]"/>
+  </edges>
+  <edges xmi:type="notation:Connector" xmi:id="_rmcvgDwjEeiDnaL1jp_puA" type="4002" source="_ef9IYClhEeikePo-S06NPw" target="_cTLscEYgEeeaV4gKFnG5PA">
+    <children xmi:type="notation:DecorationNode" xmi:id="_rmcvgzwjEeiDnaL1jp_puA" type="6001">
+      <layoutConstraint xmi:type="notation:Location" xmi:id="_rmcvhDwjEeiDnaL1jp_puA" y="40"/>
+    </children>
+    <children xmi:type="notation:DecorationNode" xmi:id="_rmcvhTwjEeiDnaL1jp_puA" type="6003">
+      <layoutConstraint xmi:type="notation:Location" xmi:id="_rmcvhjwjEeiDnaL1jp_puA" y="60"/>
+    </children>
+    <styles xmi:type="notation:FontStyle" xmi:id="_rmcvgTwjEeiDnaL1jp_puA" fontName="Ubuntu"/>
+    <element xmi:type="ecore:EReference" href="classdiagram.ecore#//Composition/composite"/>
+    <bendpoints xmi:type="notation:RelativeBendpoints" xmi:id="_rmcvgjwjEeiDnaL1jp_puA" points="[0, 0, 0, 0]$[0, 0, 0, 0]"/>
```

Comment:
ecore_diagram files have been discontinued long ago; if you want to deal with diagrams, there is a sirius-based editor for ecore files available, but we have to convert this project into a modeling project